### ISSUE TITLE
Dark area in the color mask

### ISSUE THREAD

**Issue by prophett0527** [albertpumarola/GANimation#65]
Hello, I have tried to train GANimation using my own dataset.
As you can see in the attached image, dark area(almost black) appears near the  eyes and the mouth in the color mask and the final generated image, but not in the attention mask.
![](https://i.imgur.com/zj18MTe.png)

I am wondering why the dark area appears in the color mask. 
Is it because my training dataset contains too much images which are captured in dark environment? 
If I use images with bright tone, will it help?

**Comment by albertpumarola** [comment]
Closed for inactivity
